core/src/main/scala/zio/logging/package.scala

```diff
@@ -0,0 +1,14 @@
+package zio
+
+package object logging {
+  def log(line: => String): ZIO[Logger, Nothing, Unit] =
```

Comment:
If you want to ship this right away, then rather than putting Logger in the environment, I'd make a trait `Logging` that has `def logging: Logger`. That way it can compose with other environments. Then all these can use `ZIO[Logging, ...]`.

The problem with putting `Logger` in the environment is that it can't compose easily (or at all) with other environments.

Reply:
```Logging``` trait added with all helpers